packages/components/src/components/location-info/location-info.tsx

```diff
@@ -12,6 +12,11 @@ export class LocationInfo implements ComponentInterface {
      */
     @Prop() location;
 
+    /**
+     * @description Level info.
```

Comment:
"info"? 🤔 Isn't it just a word or a phrasing?

Reply:
@ammapspeople fixed ✅

Reply:
I don't think I would understand what it was about if I was new to using the component. Can you elaborate the description to be more explaining than just "Level"?

Reply:
I tried to be more descriptive now!